src/firebase.jsx

```diff
@@ -1,21 +1,25 @@
 // Import the functions you need from the SDKs you need
 import { initializeApp } from "firebase/app";
 import { getDatabase } from "firebase/database";
+import { getStorage } from "firebase/storage";
+import { getAuth } from "firebase/auth";
 
```

Comment:
Come back here and try to use the .env
you reference the .env with VITE_SOME_....